### PR TITLE
Added "About Wharf" to the landing page

### PR DESCRIPTION
**Pull request by jilleJr**
> For Iver employees: WO0000000681896

This peice of text was requested by a coworker, some summary about Wharf. I though that we might as well add it to the documentation instead of having "Hello world."

## Preview

> https://github.com/iver-wharf/iver-wharf.github.io/blob/9161092/docs/README.md
> 
> image

### CODE REVIEW

**Review by martin-linden — DISMISSED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
-Hello world.
+Wharf adds the capabilities of self hosting and even running locally to the
+category of Continuous Integration and Continuous Deployment (CI/CD) tooling,
+while focusing on staying light weight. The only thing in your CI/CD pipeline
+that takes up significant CPU/RAM resources is your builds themselves.
+
+You deploy Wharf and grant it access to a [Kubernetes][kubernetes] cluster,
+where it will start any builds you tell it to run. You connect your favorite
+project provider, being [GitHub][github], [GitLab][gitlab], or
+[Azure DevOps][azuredevops]; or any other provider by creating your own provider
```
Comment (iverestefans) [inline docs/README.md:11]: Please, remove semicolon

Comment (jilleJr) [reply docs/README.md]: Yea it didn't really fit there

**Review by martin-linden — APPROVED** [review]
Looks good to me 🥇

**Review by iverestefans — APPROVED** [review]
LGTM!

**Pull request merged**